nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -106,9 +115,9 @@ class BtActionNode : public BT::CoroActionNode
     auto future_result = goal_handle_->async_result();
     rclcpp::executor::FutureReturnCode rc;
     do {
-      rc = rclcpp::spin_until_future_complete(node_, future_result, node_loop_timeout_);
+      rc = rclcpp::spin_until_future_complete(node_, future_result, server_timeout_);
```

Comment:
These changes don't reflect the same changes you have in the bt_service_node, make sure you're keeping these in sync

Reply:
I've reverted _bt_action_node_ as you mentioned above.

Reply:
Sorry maybe I should have been more clear:

The BTActionNode and BTServiceNode are essentially the same thing with the same purpose but one calls an action server and the other calls the service server. I wanted them both to use the same variable set, the new `server_timeout_` & all the exact changes you made to the BTServiceNode were good, but I wanted them identically reflected in BTActionNode

Reply:
Do you mean that I should use a variable like _action_server_timeout__ in _BTActionNode_ and a _service_server_timeout__ in _BTServiceNode_?  And keep using _node_loop_timeout_ in blackboard or change it to _server_timeout_?